tests/pages/components/billing-status-badge.js

```diff
@@ -0,0 +1,32 @@
+import { clickable, hasClass, isVisible, text, triggerable } from 'ember-cli-page-object';
+
+export default {
+  discountTimerBadge: {
+    scope: '[data-test-discount-timer-badge]',
+    hover: triggerable('mouseenter'),
+    click: clickable(),
+    isVisible: isVisible(),
```

Comment:
Is all of this actually required? Isn't `isVisible`/`text` etc. present by default?